pkg/proxy/iptables/proxier.go

```diff
@@ -516,6 +516,7 @@ func (proxier *Proxier) Sync() {
 func (proxier *Proxier) SyncLoop() {
 	t := time.NewTicker(proxier.syncPeriod)
 	defer t.Stop()
+	registerMetrics()
```

Comment:
Why is this in SyncLoop, rather than in app-level init (cmd) or NewProxier or something?